/**:
  ros__parameters:
    ground_truth_topic: "/robot_pose"
    wheel_odometry_topic: "/wheel_odometry"
    cmd_vel_topic: "/cmd_vel"
    ekf_state_topic: "/filtered_odometry"
    lidar_topic: "/lidar/laserscan"
    lidar_odometry_topic: "/lidar/odometry"
    lidar_sample_count: 7200
    lidar_sample_rate: 10
    imu_sensor_topic: "/imu"
    imu_sensor_update_rate: 100
    world_start_position: [-8.0, -8.0, 0.0]
    world_start_orientation: 0.0

move_action_server:
  ros__parameters:
    pose_topic: "/robot_pose"
    odometry_topic: "/lidar/odometry"